code/game/machinery/shieldgen.dm

```diff
@@ -511,22 +514,18 @@
 
 //////////////Containment Field START
 /obj/machinery/shieldwall

```

Comment:
No this is the shield itself, it should be an effect or something, not a machine with power consumption code and shit.